### PR TITLE
chore: fix line endings

### LINKED ISSUE #279: bug: cloning the git repository modifies crlf.txt
### Did you check existing issues?

- [X] I have read all the [tree-sitter docs](https://tree-sitter.github.io/tree-sitter/using-parsers) if it relates to using the parser
- [X] I have searched the existing issues of tree-sitter-cpp

### Tree-Sitter CLI Version, if relevant (output of `tree-sitter --version`)

_No response_

### Describe the bug

When cloning the git repository modifies crlf.txt.
The giattributes change the eol style of the file which modifies it.
Because of this issue the repository is always marked as dirty even when there are no change applied,
it makes it harder to use the repository in scripts/ci.

### Steps To Reproduce/Bad Parse Tree

1. git clone
2. git status

### Expected Behavior/Parse Tree

Clean source tree

### Repro

_No response_

### PR DESCRIPTION
**Pull request by ObserverOfTime**
Closes #279

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,5 +1,6 @@
 * text=auto eol=lf
-test/corpus/c/crlf.txt text eol=crlf
+
+test/corpus/c/crlf.txt eol=crlf
```
Comment (Thaodan) [inline .gitattributes:3]: Why modify line endings in gitattributes? This will change the eol of the file unless it already has crlfr line endings.

Comment (ObserverOfTime) [reply .gitattributes:3]: It **must** have crlf line endings.

**Comment by Thaodan** [comment]
Then save it has crlf line endings? Enforcing it through it attributes
makes the repository not pristine anymore e.g. when using it in ci.

It makes it impossible to clone the repository and checkout a release as
cloning will trigger gitattributes, which then triggers the file change
which then requires git checkout to revert the change from the
gitattributes.

**Comment by ObserverOfTime** [comment]
This PR makes it pristine.

**Pull request merged**